modules/lightning_features/lightning_core/modules/lightning_roles/tests/src/Kernel/ContentRoleTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\Tests\lightning_roles\Kernel;
+
+use Drupal\node\Entity\NodeType;
+use Drupal\Tests\token\Kernel\KernelTestBase;
+use Drupal\user\Entity\Role;
+
+/**
+ * @group lightning
+ * @group lightning_roles
+ */
+class ContentRoleTest extends KernelTestBase {
```

Comment:
This used to be tested functionally with Behat, but that was overkill for the functionality being tested. So I moved it into a smaller, faster kernel test.